.circleci/config.yml

```diff
@@ -77,22 +77,22 @@ anchor_verify_and_test_python: &anchor_verify_and_test_python
     - run:
         <<: *anchor_check_generation
     # Generated Logging client is broken: https://github.com/googleapis/gapic-generator/issues/2209.
-    - run:
-        name: Run Showcase Generated Unit Tests.
-        command: |
-          nox --session "unit(py='2.7')" \
-              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
-          nox --session "unit(py='3.6')" \
-              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
-        when: always
-    - run:
-        name: Run Showcase Integration Tests.
-        command: |
-          nox --session "showcase(py='2.7')" \
-              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
-          nox --session "showcase(py='3.6')" \
-              --noxfile /tmp/workspace/gapic-generator/showcase/python/nox.py
-        when: always
+    #- run:
```

Comment:
Can you file an issue and add a comment as to why this is commented out?

Reply:
And everywhere else that the Showcase tests are commented out?

Reply:
Done